ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-tests.robot

```diff
@@ -56,6 +57,36 @@ Run TestMnistPyTorchMCAD ODH test
     ...     CodeflareOperator
     Run Codeflare ODH Test    TestMnistPyTorchMCAD
 
+Run TestInstascaleMachineSet E2E test
+    [Documentation]    Run Go E2E test TestInstascaleMachineSet
+    # This particular test runs on OCP cluster with AWS
+    [Tags]  Instascale
+    ...     Tier3
+    ...     DistributedWorkloads
+    ...     CodeflareOperator
+    Skip If RHODS Is Managed
+
+    # Enable instascale to true in the codeflare operator config map
+    Enable Instascale
+
+    # Create machine set
+    Log To Console    "Create machine set ....."
+    ${result} =    Run Process    sh    ods_ci/tests/Resources/Files/machineset-template.sh
+    ...    shell=true    stderr=STDOUT
+    IF    ${result.rc} != 0
+        FAIL    Can not create machine set
+    END
+
+    # Delete the existing GPU machineset
+    # This needs to be done as part of test execution and a corresponding jira is tracked here - https://issues.redhat.com/browse/RHOAIENG-1118
+    ${result} =    Run Process    oc get machineset -n openshift-machine-api -o name | head -n1
```

Comment:
This looks suspicious. What Machine set do you delete? What is the deleted Machine set used for?